openapi/specs/v0.yaml

```diff
@@ -1130,9 +1130,9 @@ paths:
                 - $ref: '#/components/schemas/CertificatePage'
                 - $ref: '#/components/schemas/Page'
           description: Returns a page of any known Wagtail page type
-  /api/v2/pages/?fields=*&type=cms.CertificatePage:
+  /api/v2/pages/?fields=*&type=cms.certificatepage:
     get:
-      operationId: pages_?fields=*&type=cms.CertificatePage_retrieve
```

Comment:
The URLs included in the API spec required admin permission because they weren't lowercase.

Now the permission check lowercases like the filter does, but also updated these to be lowercase